packages/webapp-libs/webapp-crud-demo/src/routes/crudDemoItem/crudDemoItemList/crudDemoItemList.utils.ts

```diff
@@ -0,0 +1,7 @@
+import { CrudDemoItemListQueryQuery } from '@sb/webapp-api-client/graphql';
+
+export const isCrudDataEmpty = (data: CrudDemoItemListQueryQuery): boolean => {
+  if (data.allCrudDemoItems && data.allCrudDemoItems.edges.length <= 0) return true;
```

Comment:
This can be one-liner:
`return data.allCrudDemoItems && data.allCrudDemoItems.edges.length <= 0`
and tbh now sure about extracting this condition to a separate utils file. Looks like an overkill